test/cli/template-use/complex/in/repository/my-template/v1/src/extractor/ex-generic-v2/om-config/config.jsonnet

```diff
@@ -9,7 +9,7 @@
       ],
     },
   },
-  parameters: {
+  parameters: Input("oauthAccounts") + {
```

Comment:
CLI test, `oauthAccounts` sa pridavaju k `parameters`, co je trochu divny `layout`, ale s tym uz nic nespravime.
Z toho vyplyva, ze tu zatial nefunguje `upgrade` a nacitanie predchadzajucej hodnoty, to bude v dalsom PR.